spec/index.bs

```diff
@@ -1394,7 +1404,9 @@ interface GPUDevice : EventTarget {
     readonly attribute FrozenArray<GPUFeatureName> features;
     readonly attribute object limits;
 
-    [SameObject] readonly attribute GPUQueue defaultQueue;
+    [SameObject] readonly attribute GPUQueue queue;
+    [SameObject] readonly attribute sequence<GPUQueue> computeQueues;
+    [SameObject] readonly attribute sequence<GPUQueue> transferQueues;
```

Comment:
nit: FrozenArray instead of sequence, and I don't think SameObject is valid (on either).